test/db/cmd/cmd_ag

```diff
@@ -783,7 +772,7 @@ agraph.color_box=G1swbQ==
 agraph.delta_x=0
 agraph.delta_y=0x1
 agraph.h=0xe
-agraph.is_callgraph=false
+agraph.is_callgraph=true
```

Comment:
Why this change?

Reply:
It seems that is_callgraph controls whether the edges of the ascii graph are linked with `'>'` or `'t' | 'f'`

`ag[cC]` as Function callgraph, which I think is the correct behavior